Automattic/wp-calypso
# Fix/promote post v2 widescreen layout

## Proposed Changes

This PR fixes the layout for the new Advertising design for wide screens.

![2023-06-05 16 54 56](https://github.com/Automattic/wp-calypso/assets/1258162/8e97a594-00b0-4ff9-80e6-869125ee6ad4)

I am also removing the ListEnd element that is not included in the final designs.

## Testing Instructions

1. Download the branch locally and start Calypso
2. Go to the Advertising section (Tools->Advertising)
3. Check that the pages center the content on wide screens

## Pre-merge Checklist
